.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/libnegf/libnegf.git
 [submodule "external/scalapackfx/origin"]
     path = external/scalapackfx/origin
-    url = https://github.com/dftbplus/scalapackfx.git
+    url = https://github.com/bhourahine/scalapackfx.git
```

Comment:
Are the ScaLapackFx changes upstreamed already?

Reply:
It is now

Reply:
And just remembered to update the submodule path and hash as well...